README.rst

```diff
@@ -1,9 +1,10 @@
 biotraj
 =======
 
-WIP repository for trajectory file I/O in Biotite to solve issue
-https://github.com/biotite-dev/biotite/issues/613.
+Loading and saving raw data in trajectory formats.
```

Comment:
Maybe we could add something along these lines:
"Currently supported formats: XTC, TRR, NetCDF and DCD."